SourceX/DiabloUI/diabloui.cpp

```diff
@@ -147,7 +147,7 @@ void UiInitList(int min, int max, void(__stdcall *fnFocus)(int value), void(__st
 	for (int i = 0; i < itemCnt; i++) {
 		if (items[i].type == UI_EDIT) {
 			SDL_StartTextInput();
-			UiTextInput = items[i].caption;
+			UiTextInput = (char *)items[i].caption;

```

Comment:
Possibly we should change the type in the struct, though I seam to remember DrawArtStr complaining about it being signed originally when using it as the index for, FontTables, but possibly that is where the casting should happen.
https://github.com/diasurgical/devilutionX/blob/22a3657ee47c5f757a028b4bcd82cd056866c05c/SourceX/DiabloUI/diabloui.h#L68